src/utils/log.h

```diff
@@ -54,8 +56,6 @@
 
 #if LOG_DISABLED == 0
 
-#define IS_PRINTABLE(level) (level <= debug_level || level == L_CLI)
```

Comment:
This is still referenced in source files in the Arduino Massive SDK library, need a PR against it to switch to the new method to complete testing.